ietf/draft-ietf-calext-jscalendar-icalendar-09.txt

```diff
@@ -0,0 +1,6272 @@
+     | recurrenceOverrides | [RFC8984],    | RDATE        | Section |
+     |                     | Section 4.3.5 |              | 2.3.35  |
+     +---------------------+---------------+--------------+---------+
+     | recurrenceRules     | [RFC8984],    | RRULE        | Section |
+     |                     | Section 4.3.3 |              | 2.3.39  |
+     +---------------------+---------------+--------------+---------+
+     | start               | [RFC8984],    | DTSTART      | Section |
+     |                     | Section 4.7.2 |              | 2.3.17  |
+     +---------------------+---------------+--------------+---------+
+
+             Table 39: Properties of the TimeZoneRule object
+
+3.8.  VirtualLocation
+
+   The VirtualLocation object [RFC8984] (Section 4.2.6) converts to a
+   CONFERENCE property (Section 2.3.12).
+
+   The uri property converts to the CONFERENCE property value.  Its
+   other properties convert as follows:
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+Stepanek                   Expires 16 May 2025                 [Page 98]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+     +=============+===============+======================+=========+
+     | Name        | Reference     | Parameter (or other) | See     |
+     +=============+===============+======================+=========+
+     | description | [RFC8984],    | TBD                  |         |
+     |             | Section 4.2.2 |                      |         |
+     +-------------+---------------+----------------------+---------+
+     | features    | [RFC8984],    | FEATURE              | Section |
+     |             | Section 4.2.6 |                      | 2.3.12  |
+     +-------------+---------------+----------------------+---------+
+     | name        | [RFC8984],    | LABEL                | Section |
+     |             | Section 4.2.5 |                      | 2.3.12  |
+     +-------------+---------------+----------------------+---------+
+
+            Table 40: Properties of the VirtualLocation object
+
+4.  Updates to iCalendar
+
+4.1.  New Parameters
+
+4.1.1.  JSCALID
+
+   TBD
+
+4.2.  New Properties
+
+4.2.1.  JSCALID
+
+   TBD
+
+4.2.2.  JSPROP
+
+   TBD
+
+5.  Updates to JSCalendar
+
+5.1.  New Properties
+
+5.1.1.  completed
+
+   Name:
+      completed
+
+   Context:
+      Task
+
+   Type:
+      UTCDateTime (optional)
+
+
+
+
+Stepanek                   Expires 16 May 2025                 [Page 99]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+   Description:
+      This is the date and time when the task was completed.
+
+5.1.2.  iCalComponent
+
+   Name:
+      iCalComponent
+
+   Context:
+      Any JSCalendar object
+
+   Type:
+      ICalComponent (optional)
+
+   Description
+      This contains information about an iCalendar component that got
+      partially or fully converted to JSCalendar.  It allows to preserve
```

Comment:
"allows to preserve the name" is weird. Maybe.. "It allows for preserving the name..."